MantidPlot/src/ProjectRecovery.cpp

```diff
@@ -794,6 +799,12 @@ void ProjectRecovery::repairCheckpointDirectory() {
                 "mantid has been unable to successfully handle repair so "
                 "checkpoints may be invalid");
   }
+
+  if (vectorToDelete.size() > 0) {
+    g_log.information("Project Recovery: A repair of the checkpoint directory "
+                      "has been perfomed");
+  }
+
   bool recurse = true;
```

Comment:
Is this `bool` assigned anywhere?

Reply:
Only 2 lines below when passed to the file.remove() function. It's going to be replaced shortly.

Reply:
But if it is never assigned apart form here it is pointless being a parameter at all.

Reply:
Hence why it was removed